openrct2-worldpainter/readme.md

```
@@ -0,0 +1,80 @@
+
+## Usage
+
+To open the plugin, click on the map icon in the top left toolbar and select "WorldPainter", which opens the plugin's window. Now, if you click into the game world and hold the mouse button down, a mountain will appear in the highlighted area around the mouse cursor.
+
+Restrictions to modify terrain apply as usual. That is, the target area must be unobstructed by scenery, paths, rides and so on and you must have enough money available to perform the landscaping action. You can disable clearance checks and money through cheats if you want to circumvent this.
+
+The plugin is fully functional to use on multiplayer servers, but make sure to have landscaping permissions.
+
+## The plugin window
+
+#### Tool shape
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)